client/ayon_blender/api/plugin.py

```diff
@@ -446,7 +450,12 @@ def read(self, node) -> dict:
         if not ayon_property:
             return {}
 
-        return ayon_property.to_dict()
+        data = ayon_property.to_dict()
+        data["families"] = self.get_publish_families()
+        return data
+
+    def get_publish_families(self) -> list[str]:
```

Comment:
Can we add new families in collection phase instead? What is the point of storing them to instance metadata when they can be added during publishing...